be/api/po-app/src/main/java/cz/covid/po/api/converter/HealthCheckConverter.java

```diff
@@ -0,0 +1,39 @@
+package cz.covid.po.api.converter;
+
+import cz.covid.po.api.domain.model.HealthCheck;
+import cz.covid.po.api.domain.model.HealthCheckResult;
+import cz.covid.po.api.generated.dto.HealthCheckDto;
+import cz.covid.po.api.generated.dto.HealthCheckResultDto;
+import cz.covid.po.api.generated.dto.SymptomEnum;
+import lombok.RequiredArgsConstructor;
+import org.bouncycastle.asn1.sec.SECNamedCurves;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class HealthCheckConverter extends OneWayConverterBase<HealthCheck, HealthCheckDto> {
+    private final HealthCheckResultConverter healthCheckResultConverter;
+    private final CodebookConvertor codebookConvertor;
+
+    @Override
+    protected HealthCheckDto convertInternal(HealthCheck healthCheck) {
+        HealthCheckDto result = new HealthCheckDto();
+
+        result.setAbleToDrive(healthCheck.getAbleToDrive());
+        result.setDryCoughDuration(SymptomEnum.fromValue(healthCheck.getDryCoughDuration().name()));
```

Comment:
Pozor na to, že enum nemusí být v DB naplněn

Reply:
upraveno